### PR TITLE
add data source endpoints to OrganizationsApi

### PR DESCRIPTION
**Pull request by UnsungHero97**

### CODE REVIEW

File: src/api/OrganizationsApi.js [hunk 1]
```diff
@@ -571,6 +573,36 @@ function getOrganization(organizationId :UUID) :Promise<*> {
...
+ *
+ * @static
+ * @memberof lattice.OrganizationsApi
+ * @param {UUID} organizationId
+ * @returns {Promise<Object[]>} - a Promise that resolves with the organization data sources
+ *
+ * @example
+ * OrganizationsApi.getOrganizationDataSources("ec6865e6-e60e-424b-a071-6a9c1603d735");
+ */
+function getOrganizationDataSources(organizationId :UUID) :Promise<Object[]> {
```
Comment (katherinebernstein) [inline src/api/OrganizationsApi.js:587]: datasource is one word

Comment (katherinebernstein) [reply src/api/OrganizationsApi.js:587]: also does this API exist? I'm not seeing it

Comment (UnsungHero97) [reply src/api/OrganizationsApi.js:587]: Matthew is implementing these in the `feature/pluggable-warehouse-support` branch.

https://github.com/openlattice/openlattice/tree/feature/pluggable-warehouse-support

Comment (UnsungHero97) [reply src/api/OrganizationsApi.js:587]: Sticking with `DataSource` based on our conversation in slack.

**Review by solomontang — APPROVED** [review]

**Review by giuliacampana — APPROVED** [review]

**Pull request merged**